pkg/model/sshkey.go

```diff
@@ -37,7 +37,9 @@ func (b *SSHKeyModelBuilder) Build(c *fi.ModelBuilderContext) error {
 	t := &awstasks.SSHKey{
 		Name:      s(name),
 		Lifecycle: b.Lifecycle,
-		PublicKey: fi.WrapResource(fi.NewStringResource(string(b.SSHPublicKeys[0]))),
+	}
+	if len(b.SSHPublicKeys) == 1 {
```

Comment:
Can we make this `>= 1` (so we don't change behaviour for people that have more than 1)